src/core/crypto.c

```diff
@@ -1693,7 +1693,7 @@ QuicCryptoProcessTlsCompletion(
             memcpy(Offloads[1].PayloadIv, Path->TlsOffloadSecrets->Rx.PayloadIv, Path->TlsOffloadSecrets->Rx.PayloadIvLength);
             memcpy(Offloads[1].PayloadKey, Path->TlsOffloadSecrets->Rx.PayloadKey, Path->TlsOffloadSecrets->Rx.PayloadKeyLength);
             memcpy(Offloads[1].HeaderKey, Path->TlsOffloadSecrets->Rx.HeaderKey, Path->TlsOffloadSecrets->Rx.HeaderKeyLength);
-            if (QUIC_SUCCEEDED(CxPlatSocketUpdateQeo(Path->Binding->Socket, Offloads, 2))) {
+            if (Path->Binding != NULL && QUIC_SUCCEEDED(CxPlatSocketUpdateQeo(Path->Binding->Socket, Offloads, 2))) {
```

Comment:
Did you see a case where there was a NULL binding? I _think_ it should always be valid here.